.travis.yml

```diff
@@ -22,11 +22,18 @@ deploy:
     distributions: sdist bdist_wheel
     repo: RedHatQE/widgetastic.core
 before_install:
-  - "export PHANTOMJS_VERSION=2.1.1"
-  - "phantomjs --version"
-  - "export PATH=$PWD/travis_phantomjs/phantomjs-$PHANTOMJS_VERSION-linux-x86_64/bin:$PATH"
-  - "phantomjs --version"
-  - "if [ $(phantomjs --version) != '$PHANTOMJS_VERSION' ]; then rm -rf $PWD/travis_phantomjs; mkdir -p $PWD/travis_phantomjs; fi"
-  - "if [ $(phantomjs --version) != '$PHANTOMJS_VERSION' ]; then wget https://github.com/Medium/phantomjs/releases/download/v$PHANTOMJS_VERSION/phantomjs-$PHANTOMJS_VERSION-linux-x86_64.tar.bz2 -O $PWD/travis_phantomjs/phantomjs-$PHANTOMJS_VERSION-linux-x86_64.tar.bz2; fi"
-  - "if [ $(phantomjs --version) != '$PHANTOMJS_VERSION' ]; then tar -xvf $PWD/travis_phantomjs/phantomjs-$PHANTOMJS_VERSION-linux-x86_64.tar.bz2 -C $PWD/travis_phantomjs; fi"
-  - "phantomjs --version"
+  - "export FF_VERSION=61.0.2"
```

Comment:
Did you look at including FF/chrome with addons for travis, vs this custom version fetching?

https://docs.travis-ci.com/user/firefox
https://docs.travis-ci.com/user/chrome

Reply:
I didn't glance at this. Since I was working on other stuff, I just added a quick fix here.
But this is good idea, I noted. I'll implement it a bit later for all our wt.* repos.